public/timeline-report.js

```diff
@@ -379,45 +379,6 @@ export class TimelineReport extends StacheElement {
 customElements.define("timeline-report", TimelineReport);
 
 
-function getIssuesOfTypeAndStatus(issues, type, statuses){
-  return issues.filter( (issue)=>{
-    return issue["Issue Type"] === type && statuses.includes(issue.Status)
-  })
-}
```

Comment:
this was the only thing i found that looked like it was using the LegacyIssue type rather than IssueFields, it also wasn't being called by anything

Reply:
I'd check with @justinbmeyer if this can be removed